### ISSUE TITLE
[BITV] Border color too intense now

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/server#35457]
### Steps to reproduce

1. Run latest server master
2. Run latest Calendar main

### Expected behavior

The usual

### Actual behaviour

![Bildschirmfoto vom 2022-11-23 17-02-04](https://user-images.githubusercontent.com/1374172/203593172-fd5c5a7f-39b6-46c9-b753-f40956587024.png)

### Additional info

This change was done with https://github.com/nextcloud/server/pull/35315. @Pytal do you know if the border contrast ratio should apply for the calendar grid as well? If not, we could have lower contrast value for the calendar grid. If the contrast should be the way it is now we can close the ticket.

**Comment by jancborchardt** [comment]
Transferred to server.

@Pytal @nimishavijay @ChristophWurst the contrast is good for the input fields – but indeed for regular borders like divider lines it is much too present, like in Calendar or like below screenshot from Talk.

It would be best if we would use a separate variable like `--color-border-input` for this new value, and keep `--color-border` at the old one. cc @nimishavijay as discussed, and @AndyScherzinger @PVince81 @skjnldsv FYI


![image](https://user-images.githubusercontent.com/925062/204271266-f07870da-2678-4bd0-b36c-7bfab0ab719f.png)

**Comment by Pytal** [comment]
@michaelnissenbaum for input borders we plan on meeting the WCAG AA minimum 3:1 contrast ratio as specified under "Graphical Objects and User Interface Components" from https://webaim.org/resources/contrastchecker/

<details>
<summary>Input borders</summary>

![image](https://user-images.githubusercontent.com/24800714/204671852-a3602702-8cf6-4f19-a47c-1e002b0dffac.png)

</details>

Do the other borders shown below also need to meet this minimum contrast ratio requirement?

### Calendar

![image](https://user-images.githubusercontent.com/24800714/204671318-5a54bebd-099b-4531-afa6-f893fdf29fdd.png)

### Talk

![image](https://user-images.githubusercontent.com/24800714/204671331-71f6c743-658b-4bc3-a734-90470f1d091a.png)

**Comment by michaelnissenbaum** [comment]
Regarding calendar I'd say that the calendar lines (grid) is fine. The colours could be improved but it's not mandatory. More import is that the text within the calendar (table) meets the contrast requirements. Additionally as I can see you can have multiple calendars wich are presented in different colours within grid. Keep in mind that some user will not be able to  recognise these colours and will need alternative way - e.g. text.
Regarding Talk all elements in footer (incl. border of the input field) need to meet contrast requirements - 3:1.
<img width="733" alt="CleanShot 2022-12-01 at 08 32 07@2x" src="https://user-images.githubusercontent.com/48948959/204992969-e88830b5-3ee5-4da7-9d25-2dee523a15f5.png">

**Comment by Pytal** [comment]
Thanks @michaelnissenbaum!

We will add a separate CSS variable color for elements that need to meet the minimum 3:1 contrast requirement while keeping the existing colors used in e.g. the calendar grid and Talk sidebar border

**Comment by Pytal** [comment]
Fixed in https://github.com/nextcloud/server/pull/35563
